docs/source/modules/damping_driver.rst

```diff
@@ -0,0 +1,139 @@
+damping_driver.f90
```

Comment:
Change to a title, maybe 'Damping driver (scheme?)'.

Reply:
Changed following Penny's convention